test/testcafe/framework/page-objects/ChallengeEmailPageObject.js

```diff
@@ -8,16 +8,33 @@ export default class ChallengeEmailPageObject extends ChallengeFactorPageObject
     super(t);
   }
 
-  resendEmailView() {
-    return this.form.getElement(RESEND_EMAIL_VIEW_SELECTOR);
+  resendEmailViewText() {
+    if (userVariables.v3) {
+      return this.form.getErrorBoxText();
+    }
+    return this.form.getElement(RESEND_EMAIL_VIEW_SELECTOR).innerText;
+  }
+
+  async resendEmailExists() {
+    if (userVariables.v3) {
+      return this.form.hasErrorBox();
```

Comment:
Similar to the last comment, I would make this function name something like `hasAlertBox`